Registration/RegisterWithAzure.psm1

```diff
@@ -1214,6 +1225,19 @@ function New-RegistrationResource{
             }
         }
     } while ($currentAttempt -lt $maxAttempt)
+
+    
+    ## Set CanNotDelete Lock on Resource so the Registration resource is not deleted
+    Write-Verbose -Message "Adding CanNotDelete Lock on Registration Resource $RegistrationName..."
+    $lockNotes ="Cannot delete the Registration Resource $RegistrationName"
```

Comment:
update comment: Registration resource is needed for syndication.  Placing resource lock to prevent accidental deletion.